docs/version0.15.md

```diff
@@ -49,10 +49,12 @@
 
  OpenJ9 release 0.15.0 supports OpenJDK 8, 11, and 12.
```

Comment:
I think this should be updated as well. Either from 0.15.0 to 0.15.1, or include both numbers.

Reply:
Updated. Please merge and I will create release tags and publish